viz.yaml

```diff
@@ -356,9 +364,8 @@ publish:
     context:
       resources: ["lib_d3_js", "topojson", 
                   "map_css", "custom_css", "hashes", "projection", "styles", "map_utils", "map_states",
-                  "map_counties", "watermark_utils", "map_circles"]
+                  "map_counties", "watermark_utils", "map_circles", "static_story_figure"]
```

Comment:
if you do `export: true` in the viz block for `static_story_figure` and not include it here, does that work?

Reply:
I think "resources" here are what is added to the index.html at the top. We just need `static_story_figure` to be copied into target/images.

Reply:
to clarify, what I did here works, but I think it is a hack because I don't allow vizlab to do its thing w/ respect to getting the image path in the html (I wrote it in) and injecting things like title and alttext.